pkgs/top-level/python-packages.nix

```
@@ -10953,7 +10953,13 @@ self: super: with self; {
     cudaSupport = pkgs.config.cudaSupport or false;
   };
 
-  tensorflow-build = callPackage ../development/python-modules/tensorflow {
+  tensorflow-build =
+    # x86_64-darwin: tensorflow does not build on default 10.12 sdk
+    let call = if stdenv.hostPlatform.system == "x86_64-darwin"
+        then pkgs.darwin.apple_sdk_11_0.callPackage
+        else callPackage;
+    in
+    call ../development/python-modules/tensorflow {
```

```suggestion
  # x86_64-darwin: tensorflow does not build on default 10.12 sdk
  tensorflow-build = darwin.apple_sdk_11_0.callPackage ../development/python-modules/tensorflow {
```

This is not necessary. For linux this is a noop.